client/state/plugins/installed/reducer.js

```diff
@@ -100,6 +101,8 @@ export const plugins = withSchemaValidation( pluginsSchema, ( state = {}, action
 		case PLUGINS_RECEIVE: {
 			return { ...state, [ action.siteId ]: action.data };
 		}
+		case PLUGINS_ALL_RECEIVE:
+			return { ...state, ...action.allSitesPlugins };
```

Comment:
🤔 Do we really want to keep stale data here though? What if a site was disconnected in the meantime? Should we just return `action.allSitesPlugins` to cover such scenarios?

Reply:
Good point, I made that change.